src/applications/disability-benefits/686c-674/config/chapters/veteran-information/index.js

```diff
@@ -1,4 +1,4 @@
 import * as veteranInformation from './veteran-information/veteran-information';
-import * as veteranAddress from './veteran-address/veteran-address';
+import * as veteranAddress from './veteran-address/veteran-address.js';
```

Comment:
We don't need the `.js` on the end of this I think.

Reply:
Ugh, VS code keeps auto-populating these wierd